script/linux-build-docker

```diff
@@ -64,7 +64,7 @@ if [[ "${PLATFORM}" == "flex" ]]; then
       -e "s|EXPOSE 8080|\nADD ${DIMG} ${DIMG}\nRUN apt-get update --fix-missing -qq \&\& \\\ \n    apt-get install -qqy python \&\& \\\ \n    dpkg -i /${DIMG}\n&|" \
       "$FLEX_DIR/nginx_proxy/Dockerfile"
   DOCKERFILE_PATH="$FLEX_DIR/nginx_proxy"
-  cat "$FLEX_DIR/nginx_proxy/Dockerfile"
```

Comment:
can you move two sed before "cat" so "cat' is the final version?

There is a sed in line 63,  maybe you can add to it.

Reply:
That works. pr-e2e test was passed.